Source/WebCore/page/PointerCaptureController.cpp

```diff
@@ -634,4 +646,20 @@ void PointerCaptureController::processPendingPointerCapture(PointerID pointerId)
     m_processingPendingPointerCapture = false;
 }
 
+void PointerCaptureController::clearEventSuppression(PointerID pointerId)
+{
+    RefPtr capturingData = m_activePointerIdsToCapturingData.get(pointerId);
+    if (capturingData)
```

Comment:
nit: could be a single line
```suggestion
    if (RefPtr capturingData = m_activePointerIdsToCapturingData.get(pointerId))
```